kubernetes/samples/charts/traefik/values.yaml

```diff
@@ -1,16 +1,12 @@
-serviceType: NodePort
-service:
-  nodePorts:
-    http: "30305"
-    https: "30443"
-dashboard:
-  enabled: true
-  domain: traefik.example.com
-rbac:
-  enabled: true
-ssl:
-  enabled: true
-  #enforced: true 
-  #upstream: true
-  #insecureSkipVerify: false
-  tlsMinVersion: VersionTLS12
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```

Comment:
this file was created in 2018, so the copyright notice should be
```
# Copyright (c) 2018, 2020, Oracle Corporation and/or its affiliates.
```
this applies to all of these files that you are adding notices to.  you need to check all of them and see what years should be added to the notice

Reply:
@markxnelson shouldn't he also remove the word, "Corporation" from the copyright statement?

Reply:
Yes, you are right, thanks @rosemarymarano

Reply:
As you see, there was no copyright information in original files. Somehow er missed it.  Since I added this information first time, I added only year 2020. Anyhow updated notice with Y 21018.  I also updated few yaml files in voyager directory which had no copyright information at all.